typewriter/fixes/fix_annotate_command.py

```diff
@@ -1,13 +1,27 @@
 from __future__ import absolute_import, print_function
 
 import json
+import re
 import shlex
 import subprocess
+from lib2to3.fixer_util import does_tree_import
```

Comment:
I don't see where this is used...?

Reply:
Forgot to delete during an intermediate edit.